airbyte-integrations/connectors/source-gitlab/source_gitlab/source.py

```diff
@@ -37,6 +37,21 @@
 )
 
 
+class GitlabAuthenticator(TokenAuthenticator):
+    def __init__(self, config: Dict):
+        self.config = config
+
+    def get_auth(self) -> TokenAuthenticator:
+        private_token = self.config.get("private_token")
+        oauth_token = self.config.get("credentials")
+        if private_token:
+            # support of old config
+            return TokenAuthenticator(token=private_token)
+        if oauth_token:
+            # support of new config with oauth2.0
+            return TokenAuthenticator(token=oauth_token.get("access_token"))
```

Comment:
are there cases when access_token is not present for this value? because you used the `get` function.

Reply:
Good point, corrected.